_docs-2/administration/upgrading.md

```diff
@@ -38,7 +57,70 @@ will then need to transfer the contents of the current individual files to this
 **Warning**: Upgrading a previously encrypted instance is not supported as the implementation
 and properties have changed.
 
-## Upgrading from 1.8/9 to 2.0
+## Upgrading from 1.9/1.10/2.0 to 2.1
```

Comment:
This section header is redundant. The one above says "Upgrading from 1.10.x or 2.0.x to 2.1"
These sections need to be consolidated. As-is, it's very noisy and confusing, as the information seems spread out between the sections.

The only upgrade paths to 2.1 that we support are: 1.10.latest to 2.1.latest and 2.0.latest to 2.1.latest
Any information about versions earlier than 1.10 should simply say to upgrade to 1.10.latest first. Versions earlier than 1.10 have been EOL for nearly 2 years, I think. I don't think we need to waste time on even mentioning them here. We can direct people to the user list for help if they need to update to 1.10.

Reply:
I'll fix this in a subsequent commit